docs/UserGuide.md

```diff
@@ -28,178 +28,228 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+   - `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   - `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
 
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+   - `delete 3` : Deletes the 3rd contact shown in the current list.
 
-   * `clear` : Deletes all contacts.
+   - `clear` : Deletes all contacts.
 
-   * `exit` : Exits the app.
+   - `exit` : Exits the app.
 
-1. Refer to the [Features](#features) below for details of each command.
+1. Refer to the [Command Overview](#command-overview) below for details of each command.
 
---------------------------------------------------------------------------------------------------------------------
+---
 
-## Features
+## Interacting with DocTrack
 
-<box type="info" seamless>
+All interactions with the DocTrack platform are done through a series of **commands** entered by the user. These commands allow you to perform various **tasks**, such as adding, editing, listing, and managing persons/appointments. The commands are designed to be flexible and user-friendly, allowing **parameters** in any order, handling multiple entries for specific fields.
 
-**Notes about the command format:**<br>
+Here are some **key points** to keep in mind when using commands:
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+- You should supply all the parameters (words in `UPPER_CASE`) that are needed for a command.<br>
```

Comment:
Would "information" or "fields" be a more user-friendly language?

Reply:
Yep makes sense, I'll change it to fields across the entire user guide for consistency.